test/core/end2end/tests/connectivity.cc

```diff
@@ -65,14 +66,18 @@ static void test_connectivity(grpc_end2end_test_config config) {
   grpc_core::CqVerifier cqv(f.cq);
   child_events ce;
 
-  grpc_channel_args client_args;
-  grpc_arg arg_array[1];
+  grpc_arg arg_array[] = {
```

Comment:
These values are ignored if `fixed_reconnect_backoff_ms` is set.

Reply:
lol, no wonder my tests were failing, fixed